eeg_flow/io.py

```diff
@@ -351,10 +351,16 @@ def add_keyboard_buttons(raw: BaseRaw, eeg_stream: dict, keyboard_stream: dict)
 
             # pressed defines onset and description
             if "pressed" in event:
-                onset_lsl.append(timestamps[k])
-                description.append(button)
+                temp_onset_lsl = timestamps[k]
+                temp_description = button
             # released defines durations
             if "released" in event:
+                # values for the onset are only relevant if a released event exists
+                onset_lsl.append(temp_onset_lsl)
+                description.append(temp_description)
+                temp_onset_lsl = -1
+                temp_description = -1
```

Comment:
This was part of another PR. No need to add it here as well because now that the other PR is merged, this one enters in conflict with the branch `main` because the same files are modified. It does happen that you need to modify the same file in 2 PRs, but it should not be the same change.

I'll resolvedthe conflict and update this branch from main.

Reply:
And now that it's resolved, you can see that the changes to this file in this PR is gone ;)